src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -217,7 +217,7 @@ def dynamic_stub(channel, service, cardinalities, options=None):
   Returns:
     A face.DynamicStub with which RPCs can be invoked.
   """
-    effective_options = StubOptions() if options is None else options
+    effective_options = _EMPTY_STUB_OPTIONS if options is None else options
```

Comment:
I know, and just a few weeks ago @mehrdada was wondering what all the fuss was about!